src/index.js

```diff
@@ -428,21 +422,16 @@ class TemplateContentScript extends ContentScript {
         const detailedBill = {
           ...computedBill
         }
+        const fileurl = `${BASE_CLIENT_URL}${detailedFilepath}`
         detailedBill.filename = await getFileName(
           date,
           amount,
           currency,
-          detailed
+          detailed,
+          fileurl
```

Comment:
Why did you add the fileurl parameter here ?

Reply:
I think it was in the action, i got no good reasons 👀 
Needs to be removed 👍